src/history.ts

```diff
@@ -40,9 +40,9 @@ export class HistoryEntry extends vscode.TreeItem {
 		super(label);
 		// this command is the one when clicking the name, could be removed, or changed to viewing the gif?
 		this.command = {
-			command: 'giflens.addGif',
-			title: 'Giflens Add Gif',
-			arguments: [gifUri],
+			command: 'giflens.viewGif',
```

Comment:
at first I was like "is that a mistake? 🤔" and then "OMG he actually implemented it 😄"